src/components/TransactionStatus.svelte

```diff
@@ -8,7 +8,7 @@
   };
 </script>
 
-<div class:hidden={!showSelf}>
+<div class:hidden={!showSelf} id="transaction-status">
   <h3>Transaction status</h3>
   <ul>
     {#each statuses as status}
```

Comment:
Can the button use `on:click|preventDefault` and remove `evt.preventDefault()` above?

Reply:
yes thanks, I missed that one.